client/js/components/procedure/admin/CreateCustomFieldForm.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div data-dp-validate="createCustomFieldForm">
+    <div
+      v-if="!isOpen"
+      class="text-right mb-4">
+      <dp-button
+        data-cy="customFields:addField"
+        @click="open"
+        :text="Translator.trans('field.add')" />
```

Comment:
I opt  for a morge generic aproach and to use `add` here and `name` instead of `field.name` somewhere else

Reply:
Do you mean the translation keys?

Reply:
yes

Reply:
I didn't find `field.name`, so I changed `field.add` only 2165963e0a8dbbe1a6478dd2e19b97e25d654c24

Reply:
sorry. it was`field_name` ...

Reply:
Good idea, done in a817bac869522aea9771954ebccd2c2f405693cd